jobs/gtfs-rt-parser-v2/gtfs_rt_parser.py

```diff
@@ -197,7 +197,7 @@ def unique_filename(self):
 
 
 class RTFileProcessingOutcome(ProcessingOutcome):
-    extract_path: str
+    extract: GTFSRTFeedExtract
     aggregation: Optional[RTHourlyAggregation]
```

Comment:
AFAICT `aggregation` is literally never populated (see for example `service_alerts_outcomes` from yesterday). Can we make sure it's actually being set somewhere?

Reply:
Ah good catch, it should've been set on successes. Fixing.